k8s/ci/maya/volume/cstor/service-account.yaml

```diff
@@ -22,9 +22,6 @@ subjects:
 - kind: ServiceAccount
   name: user-service-account
   namespace: default
-- apiGroup: rbac.authorization.k8s.io
-  kind: User
-  name: system:serviceaccount:default:default
```

Comment:
Since I also made changes to the CI code, this should just test the CI setup as well right?